src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceGenerator.cs

```diff
@@ -255,7 +272,7 @@ private static MemberDeclarationSyntax GenerateIUnknownDerivedAttributeApplicati
                     .AddAttributeLists(AttributeList(SingletonSeparatedList(s_iUnknownDerivedAttributeTemplate))));
 
         // Todo: extract info needed from the IMethodSymbol into MethodInfo and only pass a MethodInfo here
-        private static IncrementalMethodStubGenerationContext CalculateStubInformation(MethodDeclarationSyntax syntax, IMethodSymbol symbol, int index, StubEnvironment environment, CancellationToken ct)
+        private static IncrementalMethodStubGenerationContext CalculateStubInformation(MethodDeclarationSyntax syntax, IMethodSymbol symbol, int index, StubEnvironment environment, ManagedTypeInfo typeKeyOwner, CancellationToken ct)
```

Comment:
TypeKeyOwner will be the deriving type for shadowing methods and cannot just be the containingType of the methods